benchmarks/ner_embeddings/project.yml

```diff
@@ -0,0 +1,278 @@
+title: "Comparing embedding layers in spaCy"
+description: |
+  This project contains the code to reproduce the results of the
+  [Multi hash embeddings in spaCy](https://arxiv.org/abs/2212.09255) technical report by Explosion.
+  The `project.yml` provides commands to download and preprocess the data sets as well as to
+  run the training and evaluation procedures. Different configuration of `vars` correspond
+  to different experiments in the report.
+
+  There are a few scripts included that were used during the technical report writing process
+  to run experiments in bulk and summarize the results. 
+  The `scripts/run_experiments.py` runs multiple experiments one after the other
+  by constructing and running `spacy project run` commands. The module
+  `scipts/collate_results.py` summarizes the results of the same trials with multiple seeds.
+  Finally, `scripts/plot_results.py` was used to produce the visualizations in the report.
+  These are all small command line apps and you can learn more about the usage as usual with the
+  `--help` flag.
+
+
+vars:
+  ner_config: "multihashembed"
+  dataset: "archaeo"
+  language: "nl"
+  vectors: "nl_core_news_lg"
+  tables_path: "tables"
+  metrics_dir: "metrics"
+  attrs: null
+  rows: "[5000, 2500, 2500, 2500]"
+  include_static_vectors: true
+  min_freq: 10
+  gpu_id: 0
```

Comment:
It seems like this project runs just fine on CPU, so to be more inclusive I would put the default to `-1`.